test/mockFunctions.spec.js

```diff
@@ -16,6 +17,18 @@ ATENÇÃO!!! Edite apenas este arquivo. Não altere os arquivos da pasta 'src'.
 
 describe('verifica as funções e os mocks', () => {
   // Crie suas mock functions aqui
+  mockFunctions.add.mockImplementation((x,y) => x + y);
+  mockFunctions.subtract.mockImplementation((x,y) => x - y);
+  mockFunctions.multiply.mockImplementation((x,y) => x * y);
+  mockFunctions.divide.mockImplementation((x,y) => x / y);
+  mockFunctions.power.mockImplementation((x,y) => Math.pow(x,y));
```

Comment:
Outra forma de se fazer é usando operador aritmético puro do js.
```suggestion
  mockFunctions.power.mockImplementation((a, b) => a ** b);
```